.github/workflows/release.yml

```
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.17]
+    steps:
+
+      - name: Set up Go  ${{ matrix.go-version }}
+        uses: actions/setup-go@v3
+        with:
+          go-version:  ${{ matrix.go-version }}
+        id: go
```

```suggestion
    steps:

      - name: Set up Go  ${{ matrix.go-version }}
        uses: actions/setup-go@v3
        with:
          go-version:  1.18
        id: go
```

There is not really any point in having a strategy with only one GO version.
And GO 1.17 will be EOL in early September when 1.19 is released so it is better to user 1.18